uktrade/data-hub-frontend
# Migrate project EditHistory fully onto React Router

## Description of change

The project Edit History page has been migrated to React Router. The deprecated `EditHistory` component has also been deleted and replaced by a new `AuditHistory` component.

I've included a couple of small changes as part of the switchover:
- All field names containing acronyms (such as UK and FDI) are now capitalised correctly
- All field names should display as they do on the project details forms
- All boolean fields now display Yes/No instead of True/False

## Test instructions

Go to the edit history page for an investment project. Everything should work correctly.

## Screenshots
The screenshot for this page is very long as it contains almost all of the possible edit history fields, open the details section below to view it.

<details summary="Screenshot">

![Screenshot 2024-02-23 at 17-09-18 Edit history - Test project renamed - Projects - Investments - DBT Data Hub](https://github.com/uktrade/data-hub-frontend/assets/36161814/760a809e-8423-4ed1-b253-31bbd24c93d2)

</details>

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
